cryptor/cryptor.go

```diff
@@ -195,6 +200,10 @@ func (encrypted *EncryptedData) computeHmac(key []byte) []byte {
 		mac.Write([]byte(encrypted.Labels[index]))
 	}
 
+	for index := range encrypted.Usages {
```

Comment:
You could also do:

    for _, value := range encrypted.Usages {
        mac.Write([]byte(value))
    }